README.md

```diff
@@ -132,12 +132,42 @@ currently, there are verification tasks for the ILP32 (32-bit) and the LP64 (64-
 
 In order to obtain verification tasks from the programs and specifications in the repository,
 a simple task-definition mechanism is used.
-For each program, the repository contains a .yml file that specifies 
-  - the subject files (the parts that a program consists of),
-  - the results (for each specification of the program, the expected outcome is indicated), and
-  - the parameters to be used for verification.
+For each program, the repository contains a .yml file that specifies the following items:
+  - `format_version`: the version of the format (a version string)
+  - `input_files`: the subject program files or directories
+    (a list of file or directory names that a program consists of)
```

Comment:
`input_files` can be either a list or a single value. In the example you give below, it is a singel value.

Proposal:

```
(a file or directory name, or a list of files or directory names that a program consists of)
```

Reply:
Thanks.

Reply:
Yes, thanks!